com.unity.netcode.gameobjects/package.json

```diff
@@ -6,6 +6,6 @@
     "unity": "2020.3",
     "dependencies": {
         "com.unity.nuget.mono-cecil": "1.10.1",
-        "com.unity.transport": "1.2.0"
+        "com.unity.transport": "1.3.0"
```

Comment:
I'd be worth to also open `testproject` and `testproject-tools-integration` Unity projects to allow them to update their manifest-lock.json and other relevant bits and pieces.

Reply:
do you mean packages-lock.json?

Reply:
done.